django/db/models/lookups.py

```diff
@@ -190,6 +197,49 @@ class FieldGetDbPrepValueIterableMixin(FieldGetDbPrepValueMixin):
     """
     get_db_prep_lookup_value_is_iterable = True
 
+    def get_prep_lookup(self):
+        prepared_values = []
+        if hasattr(self.rhs, '_prepare'):
+            # A subquery is like an iterable, but we shouldn't independently
+            # prepare its items
+            return self.rhs._prepare(self.lhs.output_field)
+        for rhs_value in self.rhs:
+            if hasattr(rhs_value, 'resolve_expression'):
+                # An expression will be handled by the database, but
+                # can co-exist alongside real values
+                pass
+            elif self.prepare_rhs and hasattr(self.lhs.output_field, 'get_prep_value'):
+                rhs_value = self.lhs.output_field.get_prep_value(rhs_value)
+            prepared_values.append(rhs_value)
+        return prepared_values
+
+    def process_rhs(self, compiler, connection):
```

Comment:
This is part of a mixin with only two uses in the standard codebase. Both overrode this method, but one of those wasn't necessary. I've removed that override and this method is now required for tests to pass.

I'm inclined to think it's better to have it here in a mixin rather than in the place I removed it from as it makes life easier for people implementing new lookups outside core.